deploy

```diff
@@ -17,6 +17,7 @@ const { ECS, Credentials } = require('aws-sdk');
       .option('container', { demandOption: true, type: 'string' })
       .option('image', { demandOption: true, type: 'string' })
       .option('image-tag', { demandOption: true, type: 'string' })
+      .option('container-env-pairs', { type: 'array', string: true })
```

Comment:
Even though your name is more precise, I would suggest something more intuitive like `env`, but I don't care too much.

Reply:
I'd keep it more explicit, since we pass 2 sets of "env" to it, the ones needed by the deployment tool itself, and the ones needed by the deployed container.
I can drop the "pairs" thought and we reach a compromise :)

Reply:
Sounds good, deal!

Reply:
🤝 getan